morph_tool/marker_helper.py

```diff
@@ -0,0 +1,200 @@
+"""Module to read/write/plot morphology markers."""
+import yaml
+from pathlib import Path
+import numpy as np
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MarkerSet:
+    """Container of several markers for a single morphology."""
+
+    def __init__(self, markers):
+        """Create list of markers.
+
+        Either a path to a yaml file, or a list of Marker objects.
+        """
+        if isinstance(markers, str):
+            with open(markers, "r") as f:
+                self._from_dicts(yaml.safe_load(f))
+        else:
+            morph_names = {marker.morph_name for marker in markers}
+            if len(morph_names) > 1:
+                raise Exception("Markers of different morphologies were provided.")
+            self.markers = markers
+            self._morph_name = morph_names.pop()
+            self.morph_path = markers[0].morph_path
+
+    def _from_dicts(self, markers):
```

Comment:
yes, not sure what  is best. To me there are only 2 ways to create a marker: from some collected data, so we use Marker for each maker we want to add, or from a direct loading of a marker.yaml (or .json if we want), which was previously saved by this API. Do you see any other relevant choices?

Reply:
I don't know, I was just thinking that we might want to gather several yaml files for example, so we could read them, then gather the lists and pass the whole list to this constructor. It would be more simple than iterating other the list, creating one Marker for each element and then passing the list of markers to the constructor.

Reply:
mmh yes, that could be better. In fact, I added a todo to append markers to a file with already some markers. So we can carry around the morph + the marker file, and add/remove stuff as we want/need. Feel free to update that part, I'm very open on that, I never know what is best to do!

Reply:
I added ctor with classmethod now